tests/aws/services/kms/test_kms.snapshot.json

```diff
@@ -2238,5 +2238,69 @@
         }
       }
     }
+  },
+  "tests/aws/services/kms/test_kms.py::TestKMS::test_re_encrypt[SYMMETRIC_DEFAULT-SYMMETRIC_DEFAULT]": {
+    "recorded-date": "09-06-2025, 12:52:58",
+    "recorded-content": {
+      "test_re_encrypt": {
+        "CiphertextBlob": "ciphertext-blob",
+        "DestinationEncryptionAlgorithm": "SYMMETRIC_DEFAULT",
+        "KeyId": "<key-id:1>",
+        "SourceEncryptionAlgorithm": "SYMMETRIC_DEFAULT",
+        "SourceKeyId": "<key-arn>",
+        "ResponseMetadata": {
+          "HTTPHeaders": {},
+          "HTTPStatusCode": 200
+        }
+      }
+    }
+  },
+  "tests/aws/services/kms/test_kms.py::TestKMS::test_re_encrypt[RSA_2048-RSAES_OAEP_SHA_256]": {
+    "recorded-date": "09-06-2025, 12:53:00",
+    "recorded-content": {
+      "test_re_encrypt": {
+        "CiphertextBlob": "ciphertext-blob",
+        "DestinationEncryptionAlgorithm": "RSAES_OAEP_SHA_256",
+        "KeyId": "<key-id:1>",
+        "SourceEncryptionAlgorithm": "RSAES_OAEP_SHA_256",
+        "SourceKeyId": "<key-arn>",
+        "ResponseMetadata": {
+          "HTTPHeaders": {},
+          "HTTPStatusCode": 200
+        }
+      }
+    }
+  },
+  "tests/aws/services/kms/test_kms.py::TestKMS::test_re_encrypt_invalid_destination_key": {
+    "recorded-date": "09-06-2025, 12:55:22",
+    "recorded-content": {
+      "invalid-destination-key-usage": {
+        "Error": {
+          "Code": "InvalidKeyUsageException",
+          "Message": "<key-arn> key usage is SIGN_VERIFY which is not valid for ReEncryptTo."
+        },
+        "message": "<key-arn> key usage is SIGN_VERIFY which is not valid for ReEncryptTo.",
+        "ResponseMetadata": {
+          "HTTPHeaders": {},
+          "HTTPStatusCode": 400
+        }
+      }
+    }
+  },
```

Comment:
nit: we can remove this since the test no longer uses snapshot validation for the response